website/content/docs/concepts/client-count.mdx

```diff
@@ -108,20 +107,26 @@ From a Vault client perspective, **_option 1_** is one client, the Master CI/CD
 The number of active clients using a Vault cluster is the total of:
 
 - active entities: identity entities that create a token via a login
-- active non-entity tokens: tokens created via a method that is not associated with an entity
+- active non-entity tokens: clients associated with tokens created via a method
+that is not associated with an entity
 
 Prior to Vault 1.6, this metric could only be measured from the audit log, using the
 `vault-advisor` tool. Starting with Vault 1.6, the number of clients per month, or for
 a contiguous sequence of months, can be measured by Vault itself.
 
+As of Vault 1.9, the total client count should always be measured using Vault itself. The
+metrics shown by the Vault UI should be regarded as the source of truth for this data. 
```

Comment:
`are the source of truth` rather than `should be regarded as the source of truth`?